src/river/river.py

```
@@ -0,0 +1,337 @@
+    QgsFeature,
+)
+from qgis.PyQt.QtCore import QVariant, QEventLoop
+from qgis.PyQt.QtWidgets import QMessageBox, QInputDialog
+from qgis.utils import iface
+from .point_selection_tool import PointSelectionTool
+from .layers.rivers_by_object_filtered import build_rivers_by_object_filtered
+from ..river.layers.basins import build_basins_layer
+from ..river.layers.rivers_and_points import build_rivers_and_points_layer
+from ..river.layers.rivers_merged import build_merged_layer
+from ..river.layers.utils import load_quickosm_layer
+from ..common import *
```

Ruff (0.8.2)

`from ..common import *` used; unable to detect undefined names

(F403)